### PR TITLE
Fix email_bulk_csv language section list

### PR DESCRIPTION
**Pull request by pmonfort**
Fix `email_bulk_csv` language list numbers

### CODE REVIEW

File: source/includes/_email_survey_csv.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ This corresponds to the language code that should be used for each row in the fi
 
 1. For multi-language accounts only
 2. To use multi-language column it's required to send `survey_settings[language]` set to `multi_language` and pass the survey template that should be used as a parameter `survey_settings[survey_template_id]`, see the [survey temaplates API](#survey-templates) to see the list of survey temaplates and available languages that have been configured in your account.
-4. All values for this column in the whole CSV file should match exactly with one of the configured language codes for the survey template that you are using.
+3. All values for this column in the whole CSV file should match exactly with one of the configured language codes for the survey template that you are using.
```
Comment (DanPitrowiski) [inline source/includes/_email_survey_csv.md]: 3. All values for this column in the CSV file should match exactly with one of the configured language codes for the survey template that you are using.

File: source/includes/_email_survey_csv.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ This corresponds to the language code that should be used for each row in the fi
 
 1. For multi-language accounts only
 2. To use multi-language column it's required to send `survey_settings[language]` set to `multi_language` and pass the survey template that should be used as a parameter `survey_settings[survey_template_id]`, see the [survey temaplates API](#survey-templates) to see the list of survey temaplates and available languages that have been configured in your account.
```
Comment (DanPitrowiski) [inline source/includes/_email_survey_csv.md]: 2. To use a multi-language column it's required to send `survey_settings[language]` set to `multi_language` and pass the survey template that should be used as a parameter `survey_settings[survey_template_id]`. Check out the [survey templates API](#survey-templates) to see the list of survey templates and available languages that have been configured in your account.

File: source/includes/_email_survey_csv.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ This corresponds to the language code that should be used for each row in the fi
 
 1. For multi-language accounts only
```
Comment (DanPitrowiski) [inline source/includes/_email_survey_csv.md]: 1. For multi-language accounts only.

**Review by DanPitrowiski — CHANGES_REQUESTED** [review]
Requested changes.

**Comment by pmonfort** [comment]
@DanPitrowiski thanks for taking the time for this! It would be nice to have your opinion about the language column names section

```
Language
   Language this column it's going to create a property

   ___language this column it's not going to create a property if the language it's multi_language

This corresponds to the language code that should be used for each row in the file.
```

https://github.com/Wootric/slate/blob/86a12050b8409f03344a045950ca89b5d06a41aa/source/includes/_email_survey_csv.md#special-columns-names

**Comment by DanPitrowiski** [comment]
@pmonfort it was easier to look at the whole section and apply some re-writes. Let me know if this helps. Yeah, I changed and improved some of my own rewrites from yesterday. Let me know if this helps.

---

### Special columns names
#### Context variables
`___context:var`

These "context" variables, if present, will be available as variables to use in variable interpolation. For example, if you pass the subject parameter as "Hello {{first_name}}!" and you add a context variable column to your CSV file (`___context:first_name`), then each email survey (each row in the file) will use the value of the `___context:first_name` as first_name and that value will be interpolated to resolve the email subject. Resulting in email subjects like "Hello John!", "Hello Anne", etc.

#### Language
Naming a column `Language` creates a property.
Naming a column `___language` does not create a property if the language is multi_language.

The language column corresponds to the language code that should be found in each row of the file.

1. This is only for multi-language accounts.
2. To use a multi-language column it's required to send survey_settings[language] set to multi_language and pass the survey template used as a parameter survey_settings[survey_template_id]. Check out the survey templates API to see the list of survey templates and available languages configured in your account.
3. All language code values for this column in the CSV should match exactly the language codes configured for the survey template you are using.

**Column Naming Reminder:**
- Integer column names should end with "_amount" (e.g.`price_amount`).
- Date column names should end with "_date" (e.g.`start_date`).

**Comment by pmonfort** [comment]
@DanPitrowiski thanks for the awesome review this helps a lot! 💪 
I have made the update, let me know if it looks good to you.

**Review by DanPitrowiski — APPROVED** [review]

**Comment by DanPitrowiski** [comment]
@pmonfort yeah it reads nicely! Thanks for those updates!

**Pull request merged**